src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -7,24 +7,99 @@
  */
 
 import React, { FunctionComponent } from 'react';
-import { EuiFlexGrid, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { formatNumber, Metric } from '../lib';
+import { EuiFlexGrid, EuiFlexItem, EuiCard, EuiStat } from '@elastic/eui';
+import { DataType, formatNumber, Metric } from '../lib';
 
 /*
- * Displays a metric with the correct format.
+ * Displays metadata for a metric.
  */
-export const MetricTile: FunctionComponent<{ metric: Metric }> = ({ metric }) => {
-  const { name } = metric;
+export const MetricCardFooter: FunctionComponent<{
+  testSubjectName: string;
+  title: string;
+  description: string;
+}> = ({ testSubjectName, title, description }) => {
   return (
-    <EuiCard
-      data-test-subj={`serverMetric-${formatMetricId(metric)}`}
-      layout="horizontal"
-      title={formatMetric(metric)}
-      description={name}
+    <EuiStat
+      data-test-subj={testSubjectName}
+      title={title}
```

Comment:
We can only change the text size of the `title` prop, so I'm using the title to render the important info (values) in the footer.